### ISSUE TITLE
Responsive Web Design

### ISSUE THREAD

**Issue by hannahackermans** [elmcip/elmcip#490]
There are a number of WCAG failures that I expect to be fixed automatically when we migrate elmcip and it becomes responsive. I am listing them here, but I am holding off on creating individual issues for those here for now. @steinmb can you take a look at this list and indicate which issues are going to be fixed after the migration and which ones require their own issue?

### WCAG Guidelines (this issue crosses several requirements)

> **Success Criterion 1.4.4 Resize text** 
> (Level AA)
> Except for [captions](https://www.w3.org/TR/WCAG21/#dfn-captions) and [images of text](https://www.w3.org/TR/WCAG21/#dfn-images-of-text), [text](https://www.w3.org/TR/WCAG21/#dfn-text) can be resized without [assistive technology](https://www.w3.org/TR/WCAG21/#dfn-assistive-technologies) up to 200 percent without loss of content or functionality.

> **Success Criterion 1.4.10 Reflow**
> (Level AA)
> Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions for:
> •	Vertical scrolling content at a width equivalent to 320 [CSS pixels](https://www.w3.org/TR/WCAG21/#dfn-css-pixels);
> •	Horizontal scrolling content at a height equivalent to 256 [CSS pixels](https://www.w3.org/TR/WCAG21/#dfn-css-pixels).
> Except for parts of the content which require two-dimensional layout for usage or meaning.

> **Success Criterion 1.4.12 Text Spacing** 
> (Level AA)
> In content implemented using markup languages that support the following [text](https://www.w3.org/TR/WCAG21/#dfn-text) [style properties](https://www.w3.org/TR/WCAG21/#dfn-style-properties), no loss of content or functionality occurs by setting all of the following and by changing no other style property:
> •	Line height (line spacing) to at least 1.5 times the font size;
> •	Spacing following paragraphs to at least 2 times the font size;
> •	Letter spacing (tracking) to at least 0.12 times the font size;
> •	Word spacing to at least 0.16 times the font size.
> Exception: Human languages and scripts that do not make use of one or more of these text style properties in written text can conform using only the properties that exist for that combination of language and script.

### Current State of ELMCIP
ELMCIP uses a narrow part of the screen. This means that the functionality is not lost when you start to resize the page. On my PC screen, I can go up to 200% as the 1.4.4 requires, but on my laptop only 120%. We have no reflow or horizontal scroll bar, so increasing beyond that point only shows the left side of the screen. 
Increasing text spacing works well in most cases, except for the text underneath “Publishers and Journals” and “Organizations” on the homepage.
